File: java-demo/java-basic-demo/src/main/java/com/kenshine/basic/_04_Annotation/test12_AnnoTypeUse/Test12.java
package com.kenshine.basic._04_Annotation.test12_AnnoTypeUse;

import java.util.ArrayList;

/**
 * @author ：kenshine
 * @date ：Created in 2022/1/9 19:41
 * @description： 类能使用的地方就能使用
 * @modified By：
 * @version: $
 */
public class Test12 {
    /**
     * @Target({ElementType.TYPE_USE})
     * 在所有类能使用的地方都能使用
     */
    public void show() throws @UseAnno("运行时异常注解") RuntimeException{
        @UseAnno ArrayList<@UseAnno String> list=new @UseAnno ArrayList<>();
        @UseAnno int num=(@UseAnno int)10L;
    }
}

File: springbootdemos07/springbootdemo689-Nashron/js/test12.js
var list = new java.util.ArrayList();
list.add("s1");
list.add("s2");
list.add("s3");
//转js数组
var jsArray = Java.from(list);
print(jsArray);                                  // s1,s2,s3
print(Object.prototype.toString.call(jsArray));  // [object Array]